test/contrib/test_socks.py

```diff
@@ -80,7 +80,8 @@ def _address_from_socket(sock):
         return socket.inet_ntop(socket.AF_INET6, ipv6_addr)
     elif addr_type == b'\x03':
         addr_len = ord(sock.recv(1))
-        return _read_exactly(sock, addr_len)
+        data = _read_exactly(sock, addr_len)
```

Comment:
should we revert this unrelated change?

Reply:
Yeah I'm going to revert all of these commits completely when I clean up this branch :)